enterprise/install-scripts/openshift.ks

```diff
@@ -1248,7 +1258,7 @@ YUM
 configure_cart_repos()
 { # add cartridge (or dependency) repo as needed
   local -A url=(
-          [jbosseap]="${jboss_repo_base}/jbeap/6/os"
+          [jbosseap]="${jboss_repo_base}/jbeap/${jbosseap_version:-6}/os"
```

Comment:
Oh, I see that `set_defaults()` sets an empty value to indicate that `configure_cart_repos()` should use `/6/` here.  That seems convoluted; couldn't `set_defaults()` simply always assign `$jbosseap_version` the value that `configure_cart_repos()` should use for the path component?